### PR TITLE
Blocks: Dropping the controls property from blocks

### PR DESCRIPTION
**Pull request by youknowriad**
In this PR, I'm updating all the blocks to use `BlockControls` in `edit` instead of the `controls` property.

related #1205

### CODE REVIEW

**Review by BE-Webdesign — APPROVED** [review]
Looks Great Riad! The gallery controls as well as the quote controls still work, so I do not believe any regressions occurred. Been a while since I looked at the blocks, and I like the new pattern where the control views are defined in the block instead of higher up.

File: blocks/library/gallery/index.js [hunk 1]
```diff
@@ -138,28 +115,29 @@ registerBlockType( 'core/gallery', {
...
-					<Toolbar controls={ [ {
-						icon: 'edit',
-						title: __( 'Edit Gallery' ),
-						onClick: () => editMediaLibrary( attributes, setAttributes ),
-					} ] } />
-				</Fill>
+		return [
+			controls,
+			focus && images.length > 1 && (
+				<InspectorControls>
```
Comment (BE-Webdesign) [inline blocks/library/gallery/index.js]: This will need a key attribute set for it I believe.

**Review by BE-Webdesign — DISMISSED** [review]

File: blocks/library/gallery/index.js (hunk 1 above)
Comment (youknowriad) [inline blocks/library/gallery/index.js]: Good catc 👍

**Comment by youknowriad** [comment]
Thanks for the review @BE-Webdesign

**Pull request merged**